gprofiler/main.py

```diff
@@ -720,19 +726,26 @@ def main() -> None:
         if client is not None and remote_logs_handler is not None:
             remote_logs_handler.init_api_client(client)
 
+        enrichment_options = EnrichmentOptions(
+            container_names=not args.disable_container_names and args.profile_api_version != "v1",
```

Comment:
> Why the and args.profile_api_version != "v1" here?

Because in "v1" we mustn't send them, anyway.

I think I understand your point - I will remove `container_names_in_protocol` and instead store the protocol version; then make those decisions based on it in `merge.py`.

Reply:
Yea, I couldn't think of any reason to specify them twice other than such optimization, but yea, that's was my point